lib/mercator.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * @typedef {Object} Point
+ * @property {Number} x
+ * @property {Number} y
+ */
+
+/**
+ * @typedef {Object} LatLng
+ * @property {Number} lat
+ * @property {Number} lng
+ */
+
+/**
+ * @typedef {Object} Tile
```

Comment:
We've got a lot of `Tile...` floating around, and they all have slightly different meanings. Could you call this `TileCoords` or `TileLocation` or something to make it super obvious this is just describing the location of a tile?

It would also be appreciated if you could go into the `geometry.js` file and set this type as the input/output for the `getTileString` and `getTileFromString` functions